web/cypress/integration/route/search-route.spec.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create and Search Route', () => {
+  beforeEach(() => {
+    // init login 
+    cy.login();
+  })
+
+  it('create route test1, test2, test3', () => {
+    //  go to route create page
+    cy.visit('/');
+    cy.contains('Route').click();
+    for (let i = 0; i < 3; i++) {
+      cy.contains('Create').click();
+      cy.get('#name').type('test' + i);
+      cy.get('#desc').type('desc' + i);
+      cy.get('#hosts_0').type('11.11.11.11');
+      cy.contains('Next').click();
+      cy.wait(400);
+      cy.get('#nodes_0_host').type('12.12.12.12', {
+        timeout: 500
+      });
+      cy.contains('Next').click();
+      cy.contains('Next').click();
+      cy.contains('Submit').click();
+      cy.contains('SubmitSuccessfully');
+      // back to route list page
+      cy.contains('Return Route List').click();
+      cy.url().should('contains', 'routes/list');
+    }
+
+  });
+
+  it('search the route', () => {
+    cy.visit('/');
+    cy.contains('Route').click();
+    // full match
+    cy.get('[title=Name]').type('test1');
+    cy.contains('查 询').click();
```

Comment:
- [ ] i18n

Reply:
not fix

Reply:
ok, I just found the APIs, refer to 

https://procomponents.ant.design/components/table#%E5%9B%BD%E9%99%85%E5%8C%96%E7%9B%B8%E5%85%B3%E7%9A%84%E9%85%8D%E7%BD%AE

Reply:
just fixed